src/FormData/MailMessageData.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace App\FormData;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class MailMessageData
+{
+    /**
+     * @var string
+     * @Assert\NotBlank(message="Veuillez choisir un sujet")
+     * @Assert\Type(type="string")
+     * @Assert\Length(max="255")
+     */
+    private $subject;
+
+    /**
+     * @var string
+     * @Assert\NotBlank(message="Veuillez saisir votre numéro de téléphone")
+     * @Assert\Type(type="string")
+     * @Assert\Length(max="25")
+     */
+    private $phone;
+
+    /**
+     * @var string
+     * @Assert\NotBlank(message="Veuillez saisir votre email")
+     * @Assert\Email
+     * @Assert\Length(max="100")
```

Comment:
There may be valid email addresses with more than 100 caracters. A quick search tells It can be up to 254